src/plays/countries-statics/CountriesStatics.jsx

```diff
@@ -1,12 +1,99 @@
 import PlayHeader from "common/playlists/PlayHeader";
 import Map from "./Map";
 import Country from "./Country";
-import { useState } from "react";
-
+import { useEffect, useRef, useState } from "react";
+import SearchAndFilter from "./SearchAndFilter";
+import { GeoContext } from "./Context";
+import MapData from "./featues.json";
+import { connectFirestoreEmulator } from "firebase/firestore";
 function CountriesStatics(props) {
   // Your Code Start below.
   const [activeGeo, setActiveGeo] = useState("ind");
+  const [showSuggestions, setShowSuggestions] = useState(false);
+  const [index, setIndex] = useState(0);
+  const [selected, setSelected] = useState("");
+  const [searchQuery, setSearchQuery] = useState("India");
+  const searchSuggestionRef = useRef(null);
+  //search query return the first 10 matching results.
+  const SearchResult = MapData.objects.world.geometries
```

Comment:
This drop-down deserves to be a component of its kind. Have you thought about packaging this and publish to npm?